wa.user.styl

```
@@ -1104,7 +1104,7 @@ OM = _omsg
         &, /._3PdFH { c: 0 0 bg5 }
 
         /// Avatars/images/new group/contact avatar/your avatar.
-        &, > img, /._2qsE7, /._1_JE6, /.JRQje, /._28-ha {
+        &, > img, /._2qsE7, /._1_JE6, /.JRQje, /._28-ha, /._3GlyB, /._8hzr9, /._1TGpC, /.gDU26 {
```

The old selectors are no longer useful/necessary and could cause conflicts in some places.

```suggestion
        &, > img, /._3GlyB, /._8hzr9, /._1TGpC, /.gDU26 {
```